stable/reloader/templates/_helpers.tpl

```diff
@@ -0,0 +1,34 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "reloader-name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" | lower -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "reloader-fullname" -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "reloader-labels.chart" -}}
+app: {{ template "reloader-name" . }}
+chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
please remove the chart and heritage, see https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#fixing-selectors

Reply:
As mentioned [here](https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata), we also need chart and heritage in main metadata labels. So I've moved `app` and `release` labels to `reloader-labels.selector` function and `chart` and `heritage` labels to `reloader-labels.chart` function. Please take a look.